server/utils/TestingUtils/DbTestingMatchers.js

```diff
@@ -0,0 +1,29 @@
+exports.matchAvailableApplications = (result) => {
+    expect.assertions(3);
+    expect(result).toHaveLength(2);
```

Comment:
I'm not sure about these magic numbers. As soon as we add new RPA Tasks to the Db this will fail, won't it?

Reply:
see [comment below](https://github.com/bptlab/ark_automate/pull/79#discussion_r560771353) please

Reply:
No it wont, because we are provisioning a local mongodb instance for the tests which we only populate with a small set of documents that are of the same format as our real database, but independent from that database